<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProtoPathDlg</class>
 <widget class="QDialog" name="ProtoPathDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select proto path</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;At the beginning. I need you to help me find the proto files.&lt;/p&gt;&lt;p&gt;e.g: &lt;/p&gt;&lt;p&gt;Root path is &lt;span style=&quot; text-decoration: underline;&quot;&gt;E:/&amp;lt;Your_project&amp;gt;/Depend/Common/include/ProtoCommon/Proto&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Load path is &lt;span style=&quot; text-decoration: underline;&quot;&gt;E:/&amp;lt;Your_project&amp;gt;/Depend/Common/include/ProtoCommon/Proto/Client&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Proto root path:   </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editRootPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnRootPath">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Proto path to load:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editLoadPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnLoadPath">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ProtoPathDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
